unpackaged/config/qa/objects/Contact.object

```diff
@@ -68,29 +47,4 @@
         <type>Text</type>
         <unique>true</unique>
     </fields>
-    <webLinks>
```

Comment:
These are the custom buttons we just added to work around namespacing issues in the child packages. Do they now exist in another flow/task?

Reply:
Yes, I moved them to unpackaged/config/trial. This is one thing I wasn't sure about because they aren't needed for managed NPSP (but I did use namespace tokens so at least they will work there). But if we move them back to unpackaged/config/qa we don't currently have a good way to add them to the page layouts without duplicating a bunch of layout metadata, so I figured this is the best way for now.